### PR TITLE
Indonesian Translation Support

### PR DESCRIPTION
**Pull request by IdaeChop**
Indonesian Translation V1

### CODE REVIEW

**Comment by Nv7-GitHub** [comment]
Probably pretty out-of-date now, need to update ur fork to match mine

**Comment by IdaeChop** [comment]
> Probably pretty out-of-date now, need to update ur fork to match mine

I've fetched upstream

**Comment by Nv7-GitHub** [comment]
Is this up-to-date? (like does it have all the things translation files are supposed to have)

File: eod/translation/languages/id.json [hunk 1]
```diff
@@ -0,0 +1,49 @@
+{
```
Comment (Nv7-GitHub) [inline eod/translation/languages/id.json:1]: Can you rename this to `in.json` to match https://www.sitepoint.com/iso-2-letter-language-codes/ ?

**Review by Nv7-GitHub — CHANGES_REQUESTED** [review]

**Comment by Nv7-GitHub** [comment]
There were some more updates, could you add them?

File: eod/translation/languages/id.json (hunk 1 above)
Comment (IdaeChop) [reply eod/translation/languages/id.json:1]: I set it as id because the code for Indonesia in ISO-3166 alpha-2 is ID, but I probably should change it yes.

**Comment by IdaeChop** [comment]
Updated Indonesian Support

**Comment by IdaeChop** [comment]
> Updated Indonesian Support

It is now done

**Review by IdaeChop — COMMENTED** [review]
done

File: eod/translation/languages/in.json [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+  "PollNoExist": "Saranan tidak nyata!",
+  "CalculatingPing": "Dikalkulasikan...",
+
+  "SentWordCloud": "Dikirim awan kata ke DM!",
+  "WordCloudElem": "Awan kata untuk **%s**:",
+  "WordCloudDimensionsTooLow": "Besar minimum sebuah awan kata adalah 1x1!",
+  "WordCloudDimensionsTooHigh": "Besar maksimal sebuah awan kata adalah 4096x4096!",
+
+
+  "$schema": "../schema.json"
```
Comment (Nv7-GitHub) [inline eod/translation/languages/in.json:207]: Seems like you're missing `NewComboNews` and `NewElemNews`

**Review by Nv7-GitHub — CHANGES_REQUESTED** [review]

File: eod/translation/languages/in.json (hunk 2 above)
Comment (IdaeChop) [reply eod/translation/languages/in.json:207]: I see, alright then.

**Review by Nv7-GitHub — APPROVED** [review]
Seems good!

**Pull request merged**